### PR TITLE
New package: khmer-fonts-ttf-5.0

### PR DESCRIPTION
**Pull request by zdtcd**

### CODE REVIEW

File: srcpkgs/khmer-fonts-ttf/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'khmeros-fonts-ttf'
```
Comment (pullmoll) [inline srcpkgs/khmer-fonts-ttf/template]: `# Template file for 'khmer-fonts-ttf'`

File: srcpkgs/khmer-fonts-ttf/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+pkgname=khmer-fonts-ttf
+version=5.0
+revision=1
+archs=noarch
+hostmakedepends="unzip"
+depends="font-util"
+short_desc="Fonts for Khmer and Latin script"
+maintainer="Doan Tran Cong Danh <congdanhqx@gmail.com>"
+license="LGPL"
```
Comment (pullmoll) [inline srcpkgs/khmer-fonts-ttf/template]: `license="LGPL-2.1-or-later"` probably.

Comment (zdtcd) [reply srcpkgs/khmer-fonts-ttf/template]: Well, I'm not sure about their license.
I think this is their homepage: http://www.khmeros.info/?q=en/fonts

**Comment by Chocimier** [comment]
Please set `create_wrksrc=yes` instead of do_extract.
They didn't said what LGPL version, so we can use any.

**Comment by zdtcd** [comment]
> Please set `create_wrksrc=yes` instead of do_extract.
> They didn't said what LGPL version, so we can use any.

Yeah, from LGPL, if they don't specify which version of LGPL, we can use whichever we like.
2.1 would be a sensible choice

**Comment by pullmoll** [comment]
@zdtcd you need to set workdir to the name of the path contained in the zip. Here's a working template:
```
# Template file for 'khmer-fonts-ttf'
pkgname=khmer-fonts-ttf
version=5.0
revision=1
wrksrc="All_KhmerOS_${version}"
archs=noarch
hostmakedepends="unzip"
depends="font-util"
short_desc="Fonts for Khmer and Latin script"
maintainer="Doan Tran Cong Danh <congdanhqx@gmail.com>"
license="LGPL-2.1-or-later"
homepage="http://www.khmeros.info/?q=en/fonts"
distfiles="${SOURCEFORGE_SITE}/khmer/Fonts%20-%20KhmerOS/KhmerOS%20Fonts%20${version}-%20LGPL%20Licence/All_
KhmerOS_${version}.zip"
checksum=62fec2273016fb6e69b18635e696fd2c91953af9cbe757b341721aec2232432e
font_dirs="/usr/share/fonts/TTF"

do_install() {
        for f in ./*.ttf; do
                vinstall "$f" 644 usr/share/fonts/TTF
        done
}
```

BTW one of the fonts is `KhmerOS .ttf` - note the blank between the name a the `.ttf`.

**Comment by zdtcd** [comment]
> BTW one of the fonts is `KhmerOS .ttf` - note the blank between the
> name a the `.ttf`.

Should we strip the blank or keep it as is?

-- 
Danh

**Comment by pullmoll** [comment]
I'd say leave it as it is. Just remember the fact in case it's causing problems for somebody later on... I could imagine a GUI input field where you enter a font name, and which strips leading / trailing blanks, but I don't know of such an application :)

**Pull request merged**